release/ray_release/scripts/build_pipeline.py

```diff
@@ -163,7 +156,9 @@ def main(
                     test,
                     test_collection_file,
                     run_id=run_id,
-                    report=report,
+                    # Always report performance data to databrick. Since the data is
+                    # indexed by branch and commit hash, we can always filter data later
```

Comment:
I am a bit worry it will generate a lot of noise data.

maybe make it controllable by a env var or something?

why do people want to have results from other branches also in databricks?

Reply:
My main worry is on the volume of data instead of noise. But given the volume of running release tests on PR, I think that's ok.

The current data schema requires filtering based on branch anyway so it wouldn't affect existing tools.

Reply:
Schema: https://github.com/ray-project/ray/blob/master/release/ray_release/reporter/db.py#L29

Reply:
also databricks is ok for tracking a trend, but feels like not the right tool to use for tracking results from a PR?

Reply:
i think they are interested in performance data which is log part of the schema (not the test status)